package-testing/Rakefile

```diff
@@ -26,6 +27,23 @@ namespace :acceptance do
     # to use this beaker suite outside of Puppet Inc, set BUILD_SERVER to the hostname of a build server that uses the directory hierarchy beaker assumes
     ENV['BUILD_SERVER'] ||= 'builds.delivery.puppetlabs.net'
 
+    if ENV['SHA'] && ENV['SHA'] == 'latest'
+      page = Nokogiri::HTML(open("https://#{ENV['BUILD_SERVER']}/pdk"))
```

Comment:
You can use this url to sort by date descending and avoid needing to sort below: http://builds.delivery.puppetlabs.net/pdk/?C=M&O=D

Reply:
Ah, nice! Will update